packages/components/src/templates/dialog.stories.ts

```diff
@@ -0,0 +1,150 @@
+import '../solid-components';
+import { html } from 'lit-html';
+
+export default {
+  tags: ['!dev'],
+  title: 'Templates/Dialog',
+  parameters: {
+    chromatic: { disableSnapshot: true },
+    design: {
+      type: 'figma',
+      url: ''
+    }
+  }
+};
+
+/**
+ *
+ */
+
+export const Default = {
+  name: 'Dialog with Button Group',
```

Comment:
How about the mobile screen that shows up automatically when the viewport is smaller than 414px.

It has smaller headline and paddings and the buttons are streched and vertically stacked.

Could we show that in the storybook template somehow?
I cannot switch to a mobile view in the templates.

Reply:
It's not possible to create stories specifically to show responsive behaviour, this needs to be done by resizing the window. I've made some changes to the template to overcome a [bug](https://github.com/solid-design-system/solid/issues/1325) we discovered in the dialog